Microsoft.WindowsAzure.Storage/CMakeLists.txt

```diff
@@ -15,7 +15,33 @@ option(BUILD_SAMPLES "Build sample codes" OFF)
 if(UNIX)
   find_package(Boost REQUIRED COMPONENTS log log_setup random system thread locale regex filesystem chrono date_time)
   find_package(Threads REQUIRED)
-  find_package(OpenSSL REQUIRED)
+  if(APPLE AND NOT OPENSSL_ROOT_DIR)
+    # Prefer a homebrew version of OpenSSL over the one in /usr/lib
```

Comment:
I tend to avoid `brew link --force` I've broken my system in the past with it. Even homebrew recommend against it when you install keg-only packages.